rpm/topology.spec

```diff
@@ -0,0 +1,33 @@
+Summary: osg-notify tool
+Name: osg-notify
+Version: 1.0.0
+Release: 1%{?dist}
+Source0: topology-%{version}.tar.gz
+License: Apache 2.0
+BuildArch: noarch
+Url: https://github.com/opensciencegrid/topology/
+Provides: osg-notify = %{version}-%{release}
+Requires: python-gnupg
+Requires: python-requests
+
+%description
+A simple tool for generating notification emails to the OSG
+
+%prep
+tar xzf %{SOURCE0}
+cd topology-%{version}
+
```

Comment:
I know about this macro, but I think it will use the tag "name" to name the tarball. In this case the tarball is called topology-1.0.0.tar.gz insetad of osg-notify-1.0.0.tar.gz, that's why I hardcoded it

Reply:
I think I'm wrong about what I described above, will test your suggestion and see

Reply:
I managed to use the macro by passing the "-n" option